docs/design/diagrams/managed_subscribe_module_sequence.puml

```diff
@@ -0,0 +1,65 @@
+@startuml Managed Subscribe Module
+title Managed Subscribe Module Sequence
+participant DTProvider
```

Comment:
Maybe replace `DTProvider` and `DTConsumer` with `Digital Twin Provider` and `Digital Twin Consumer`  respectively to be consistent with the other sequence diagrams in Ibeji.

Reply:
changed